### PR TITLE
Compatibility box position & width

### PR DESCRIPTION
**Pull request by Michaelsy**
Fixes https://github.com/splitbrain/dokuwiki/issues/2407

### CODE REVIEW

**Comment by Klap-in** [comment]
Hi!
What does your change with the fontsize?

**Comment by Klap-in** [comment]
The 'column' with the compatibility info is now wider than the 'column' with the description. 
The descriptions contains often longer text, which is compressed to more lines...

compatibilitybox-relayout

@selfthinker could you maybe explain why the ul element is truncked at the edge of the page content?
Is it doable to let the ul hover over that edge?
other alternative: Maybe position the box a step to the left. (means it is also not nice aligned anymore with collapsed compatibility..)

**Comment by Michaelsy** [comment]
>  could you maybe explain why the ul element is truncked at the edge of the page content?

This is caused by:

    .dokuwiki div.page {
        [...]
        overflow: hidden;
        [...]
    }

**Comment by Klap-in** [comment]
Ah, good! That is difficult to override...

File: style.css [hunk 1]
```diff
@@ -113,11 +113,12 @@
     top: 0;
     left: 0;
     box-shadow: 0 5px 5px #999;
-    width: 300px;
+    width: 21.2em;
```
Comment (phy25) [inline style.css:116]: Remove this width rule (and keep it the same as the state not in hover).

**Comment by Michaelsy** [comment]
Hello @phy25 , thank you for taking care of my PR.

Unfortunately, I have practically no experience with the creation and further handling of PRs. Back when I created this PR, I had the idea to familiarize myself with it. In the meantime, however, I have given up this idea for several reasons.

I've had it before that my attempt to change a PR and adapt it to the wishes mentioned ended in a complete disaster. In the end I could only ending the problem by canceling the PR.

What I actually want to say: Please make the changes you want if you would like to do so.

Otherwise I withdraw this PR so that it does not end in an accident again.

Thank you again!

**Comment by Michaelsy** [comment]
Just one more remark:

Meanwhile I wonder if it wouldn't be better to solve the problem on the HTML respectively PHP level. In my opinion, the error that basically exists here is that too many elements are unnecessarily grouped horizontally next to each other. The Compatibility box could also be placed below the header. This would eliminate all space problems at once.

In this sense I wonder if this PR really makes sense at all, or if it approaches the problem from the wrong side.

Sorry, just my 5 cents

**Comment by Michaelsy** [comment]
I close this PR as it has been replaced by https://github.com/splitbrain/dokuwiki-plugin-pluginrepo/pull/130

**Pull request closed without merging**